main.py

```
@@ -1134,0 +1,1230 @@
+        Args:
+            links_file: Path to the file containing links to scrape
+            output_dir: Directory to save markdown files
+            save_chunks: Whether to save chunks for RAG
+            chunk_dir: Directory to save chunks (defaults to output_dir/chunks)
+            chunk_format: Format to save chunks (json or jsonl)
+
+        Returns:
+            List of successfully scraped URLs
+        """
+        # Read links from file
+        with open(links_file, "r", encoding="utf-8") as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)